apps/ledger-live-mobile/src/newArch/features/Web3Hub/components/ManifestsList/index.tsx

```diff
@@ -54,6 +55,11 @@ export default function ManifestsList({
 
   const goToApp = useCallback(
     (manifestId: string) => {
+      track("web3hub_app_clicked", {
+        component: `ManifestsList`,
+        page: "Web3HubMain",
+        app: manifestId,
+      });
```

Comment:
What I said on the other track events that could be replaced by a simple `TrackScreen` could also be applied for this as it would gives us the full flow from each screen with minimal events on the dashboard